dockers /Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM amazonlinux:2023
+RUN yum update 
+WORKDIR APP
+RUN yum install wget -y
+RUN mkdir -p /usr/share/man/man1/
+RUN wget https://download.java.net/java/GA/jdk17/0d483333a00540d886896bac774ff48b/35/GPL/openjdk-17_linux-x64_bin.tar.gz
+RUN yum install tar -y
+RUN yum install gzip -y
+RUN tar xvf openjdk-17_linux-x64_bin.tar.gz
```

Comment:
Is OpenJDK 17 available in the package repositories?

You're also using multiple RUN statements which should be concatenated. E.g.:
```
RUN yum install -y \
    wget \
    tar \
    gzip \
RUN mkdir -p /usr/share/man/man1/ && \
    wget https://download.java.net/java/GA/jdk17/0d483333a00540d886896bac774ff48b/35/GPL/openjdk-17_linux-x64_bin.tar.gz && \
    tar xvf openjdk-17_linux-x64_bin.tar.gz
```